src/main/kotlin/com/sourcegraph/cody/CodyToolWindowContent.kt

```diff
@@ -246,13 +247,10 @@ class CodyToolWindowContent(private val project: Project) : UpdatableChat {
 
     // Loop on recipes and add a button for each item
     for (recipe in recipes) {
-      if (recipe.id == null || recipe.title == null) {
-        continue
-      }
-      val recipeButton = createRecipeButton(recipe.title!!)
+      val recipeButton = createRecipeButton(recipe.title)
       recipeButton.addActionListener {
         GraphQlLogger.logCodyEvent(project, "recipe:" + recipe.id, "clicked")
-        sendMessage(project, recipe.title!!, recipe.id!!)
+        sendMessage(project, recipe.title, recipe.id)
```

Comment:
I see that a few fields are no longer nullable thanks to these changes. this is great 🚀